packages/loader/src/__tests__/index-test.ts

```diff
@@ -0,0 +1,94 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { Stats } from 'webpack';
+import compiler from './__helpers__/compiler';
+import * as fromQuery from '@gql2ts/from-query';
+import buildDeclaration from '../buildDeclaration';
+import { PossibleSchemaInput } from '../../../types/node_modules/@gql2ts/util';
+import { ITypeMap, IProvidedOptions } from '../../../from-query/node_modules/@gql2ts/types';
```

Comment:
These two paths above need to be fixed

Reply:
(this is why tests are broken)

Reply:
🤦‍♂️ auto imported use these paths, should've caught this!